File: app/src/main/java/in/example/rahul/vegcartpro/SplashActivity.java
package in.example.rahul.vegcartpro;

import android.content.Intent;
import android.os.CountDownTimer;
import android.support.v7.app.ActionBar;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

public class SplashActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_splash);
        ActionBar actionBar=getSupportActionBar();
       // actionBar.hide();
        new CountDownTimer(1000,1000){
            @Override
            public void onTick(long millisUntilFinished){

            }

            @Override
            public void onFinish() {
                Intent intent=new Intent(SplashActivity.this, HomeActivity.class);
                startActivity(intent);
                finish();
            }
        }.start();
    }
}

File: app/src/main/java/in/rahulkhatri/vegcartpro/activity/SplashActivity.kt
package `in`.rahulkhatri.vegcartpro.activity

import `in`.rahulkhatri.vegcartpro.R
import android.content.Intent
import android.os.Bundle
import android.os.CountDownTimer
import androidx.appcompat.app.AppCompatActivity

class SplashActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_splash)
//   ActionBar actionBar=getSupportActionBar();
// actionBar.hide();
        object : CountDownTimer(1500, 1000) {
            override fun onTick(millisUntilFinished: Long) {}
            override fun onFinish() {
                val intent = Intent(this@SplashActivity, HomeActivity::class.java)
                startActivity(intent)
                finish()
            }
        }.start()
    }
}